pilot/main.py

```diff
@@ -87,4 +94,3 @@ def init():
     finally:
         if run_exit_fn:
             exit_gpt_pilot(project, ask_feedback)
-        sys.exit(0)
```

Comment:
why is this removed?

Reply:
Unrelated change that crept in.

This call to `sys.exit()` has no effect, since there's nothing after "finally", and any exceptions that might have been raised are catched and handled before.